### PR TITLE
Add mira-omf, las-geoh5, lasio and simpeg-drivers

### PR DESCRIPTION
**Pull request by SophieCurinier**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/las-geoh5/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MiraGeoscience/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1c20e6bb021516ef89de8551dc1f47f944bc98ea83d3da393d13e71469add43d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m conda config --add channels kinverarity && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Comment (SophieCurinier) [inline recipes/las-geoh5/meta.yaml:14]: Hi @conda-forge/help-python, could you help me ? I really need to add another conda channel `kinverarity`. Have you any idea, how to do it?

Comment (ocefpaf) [reply recipes/las-geoh5/meta.yaml:14]: We do not support mixing with third party channel b/c we don't know and cannot verify what is in there. You must build everything you need in conda-forge.

Comment (SophieCurinier) [reply recipes/las-geoh5/meta.yaml]: Thank you @ocefpaf . I'm gonna publish `lasio` package on conda-forge. Thank you

**Comment by SophieCurinier** [comment]
@sebhmg Can you confirm that you are ready to be listed as a maintainer?

File: recipes/las-geoh5/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - pip
+
+about:
+  home: https://www.mirageoscience.com/mining-industry-software/python-integration/
+  summary: 'Import/Export LAS files to/from geoh5 format.'
+  description: |
+    Import/Export LAS files to/from geoh5 format.
+
+    This package allows for import and export of LAS files to and from a drillhole group. There is a module each for import and export operations. Each of these includes a driver and a ui.json that parameterizes the driver and renders a UI for use within Geoscience ANALYST Pro. Read on to learn how to install las-geoh5 and get started importing and exporting LAS files.
+    Warning: This package is not maintain by the owner of the original package.
```
Comment (sebhmg) [inline recipes/las-geoh5/meta.yaml:51]: **issue:** this warning does not belong here, but to the lasio recipe

File: recipes/lasio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/kinverarity1/lasio
+  summary: 'Read/write well data from Log ASCII Standard (LAS) files'
+  description: |
+    This is a Python 3.7+ package to read and write Log ASCII Standard (LAS) files, used for borehole data such as geophysical, geological, or petrophysical logs.
+    It's compatible with versions 1.2 and 2.0 of the LAS file specification, published by the Canadian Well Logging Society. Support for LAS 3 is being worked on.
```
Comment (sebhmg) [inline recipes/lasio/meta.yaml:50]: Warning about the recipe not being maitained by the author of the Python package goes here, with some fromatting
```md
`**WARNING:** ...`
```

or more with panel, also maybe more a note than a warning as we do not want to discourage using this package:
```md
<div style="background-color: #e6f7ff; padding: 10px;">
  <strong>Note:</strong>
  Maintainers of this recipe are not the authors of the lasio Python package itself.</p>
</div>
```

image

File: recipes/lasio/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/kinverarity1/lasio
+  summary: 'Read/write well data from Log ASCII Standard (LAS) files'
+  description: |
+    This is a Python 3.7+ package to read and write Log ASCII Standard (LAS) files, used for borehole data such as geophysical, geological, or petrophysical logs.
```
Comment (sebhmg) [inline recipes/lasio/meta.yaml:49]: have a white line to force a line break
```suggestion
    This is a Python 3.7+ package to read and write Log ASCII Standard (LAS) files, used for borehole data such as geophysical, geological, or petrophysical logs.

```

File: recipes/simpeg-drivers/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    - h5py >=3.2.1,<4.0.0
+    - matplotlib-base >=3.7.1,<3.8.0
+    - mkl 2022.1.*
+    - pandas >=2.2.1,<2.3.0
+    - pydantic >=2.5.2,<2.6.0
+    - pymatsolver >=0.2.0,<0.3.0
+    - scikit-learn >=1.4.0,<1.5.0
+    - tqdm >=4.66.1,<5.0.0
+    - tbb 2021.12.*
+    - python-tzdata 2023.4.*
```
Comment (sebhmg) [inline recipes/simpeg-drivers/meta.yaml:49]: **question:** does it need to repeat indirect dependencies, such as h5py which its already required by geoh5py?

File: recipes/simpeg-drivers/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+    - pillow >=10.3.0,<10.4.0
+    - fsspec 2022.*
+    - geoana >=0.4.0,<0.5.0
+    - h5py >=3.2.1,<4.0.0
+    - matplotlib-base >=3.7.1,<3.8.0
+    - mkl 2022.1.*
+    - pandas >=2.2.1,<2.3.0
+    - pydantic >=2.5.2,<2.6.0
+    - pymatsolver >=0.2.0,<0.3.0
```
Comment (sebhmg) [inline recipes/simpeg-drivers/meta.yaml:45]: misses pydiso = "~0.0.3"

for simpeg
As its version is not constrained by simpeg, needs to specify it here

File: recipes/mira-omf/meta.yaml [hunk 7]
```diff
@@ -52,7 +52,7 @@ about:
   summary: 'API library for Open Mining Format, a new standard for mining data backed by the Global Mining Standards & Guidelines Group.'
   description: |
     API library for Open Mining Format, a new standard for mining data backed by the Global Mining Standards & Guidelines Group.
-    Warning
+    **Note:**
```
Comment (sebhmg) [inline recipes/mira-omf/meta.yaml:55]:
```suggestion

    **Note:**
```

File: recipes/simpeg-drivers/meta.yaml [hunk 8]
```diff
@@ -45,8 +48,7 @@ requirements:
     - pymatsolver >=0.2.0,<0.3.0
     - scikit-learn >=1.4.0,<1.5.0
     - tqdm >=4.66.1,<5.0.0
-    - tbb 2021.12.*
-    - python-tzdata 2023.4.*
+    - pydiso >=0.0.3,<0.1.0
```
Comment (sebhmg) [inline recipes/simpeg-drivers/meta.yaml:51]: **suggestion (readability):** preserve alphabetical order

File: recipes/simpeg-drivers/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.10, <3.11
```
Comment (ocefpaf) [inline recipes/simpeg-drivers/meta.yaml:20]:
```suggestion
    - python >=3.10,<3.11
```

**Pull request merged**